docs/sdk/clients.md

```diff
@@ -97,6 +97,12 @@ gaiacli send \
 The `--amount` flag accepts the format `--amount=<value|coin_name>`.
 :::
 
+::: tip Note
+You may want to cap the maximum gas that can be consumed by the transaction via the `--gas` flag.
+If set to 0, the gas limit will be automatically estimated.
+Gas estimate might be inaccurate as state changes could occur in between the end of the simulation and the actual execution of a transaction, thus an adjustment is applied on top of the original estimate in order to ensure the transaction is broadcasted successfully. The adjustment can be controlled via the `--gas-adjustment` flag, whose default value is 1.2.
```

Comment:
Should we write up into an issue to make this configurable per tx type in a config somewhere? The reason for considering it would be that a bank send tx probably is going to be most common tx, and also shouldn't have an increased cost due to other state changes, so it shouldn't have the same adjustment as staking txs.

Reply:
Makes sense. Please fork it into a new issue :+1:

Reply:
Yeah, this is definitely something to consider because the user pays for the allocated gas, not the consumed gas.

Reply:
#2140